src/System.Net.Security/src/System/Net/Windows/SSPISecureChannelType.cs

```diff
@@ -0,0 +1,790 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.ComponentModel;
+using System.Globalization;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Principal;
+
+namespace System.Net
+{
+    // For SSL connections:
+    internal partial class SSPISecureChannelType : SSPIInterface
```

Comment:
For NegotiateStream it looks like this is better back the way it was. The PAL layer should probably be set higher than *SSPI* classes that are Windows specific.